designcompose/src/androidTest/kotlin/com/android/designcompose/JniFetchTests.kt

```diff
@@ -38,9 +38,9 @@ class JniFetchTests {
         try {
             LiveUpdateJni.jniFetchDoc("DummyDocId", requestJson)
             Assert.fail("Should have thrown an exception")
-        } catch (exception: Exception) {
+        } catch (exception: AccessDeniedException) {
```

Comment:
What happens if a different exception is thrown here? Will the test fail?

Reply:
Not sure, I should find the right "assert that exception occurred" assertion.

Reply:
I think that the test will fail because any other kind of exception won't be caught. Tim tells me that Kotlin makes all exceptions unchecked.